tests/codegen/filename.rs

```diff
@@ -0,0 +1,18 @@
+// CHECK: filename
+
```

Comment:
In addition to the filecheck tests, you should add a couple of blessed tests. Add `tests/codegen-output/`, teach bootstrap to codegen all the `.rs` files and assert their output matches a `.c` of the same name in the same directory. And then add a `--bless` argument to bootstrap that will update the `.c` file.

Don't use that for complex tests where the MIR output is likely to change. But this would be good for simple tests like this one, and also is nice to show what the expected output is.

Reply:
Also - these tests should get a comment about what they are intended to test. I'm not sure what this one is supposed to do.